packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -64,21 +67,80 @@ const demoTable = new db.Table(stack, 'DemoTable', {
 
 const demoDS = api.addDynamoDbDataSource('demoDataSource', 'Table for Demos"', demoTable);
 
-// Resolver for the Query "getDemos" that scans the DyanmoDb table and returns the entire list.
+// Resolver for the Query "getDemosDynamo" that scans the DyanmoDb table and returns the entire list.
 demoDS.createResolver({
   typeName: 'Query',
-  fieldName: 'getDemos',
+  fieldName: 'getDemosDynamo',
   requestMappingTemplate: MappingTemplate.dynamoDbScanTable(),
   responseMappingTemplate: MappingTemplate.dynamoDbResultList(),
 });
 
-// Resolver for the Mutation "addDemo" that puts the item into the DynamoDb table.
+// Resolver for the Mutation "addDemoDynamo" that puts the item into the DynamoDb table.
 demoDS.createResolver({
   typeName: 'Mutation',
-  fieldName: 'addDemo',
+  fieldName: 'addDemoDynamo',
   requestMappingTemplate: MappingTemplate.dynamoDbPutItem(PrimaryKey.partition('id').auto(), Values.projecting('demo')),
   responseMappingTemplate: MappingTemplate.dynamoDbResultItem(),
 });
+
+// Create username and password secret for DB Cluster
+const secret = new DatabaseSecret(stack, 'AuroraSecret', {
+  username: 'clusteradmin',
+});
+
+// Create the DB cluster, provide all values needed to customise the database.
+const cluster = new DatabaseCluster(stack, 'AuroraCluster', {
+  engine: DatabaseClusterEngine.auroraMysql({
+    version: AuroraMysqlEngineVersion.VER_2_07_1,
+  }),
+  masterUser: {
+    username: 'clusteradmin',
+  },
+  clusterIdentifier: 'db-endpoint-test',
+  defaultDatabaseName: 'demos',
+});
+
+// Build a data source for AppSync to access the database.
+const rdsDS = api.addRdsDataSource('rds', 'The rds data source', cluster, secret);
+
+// Set up a resolver for an RDS query.
```

Comment:
nit: i dont think the addition of rds data source is necessary

the README functions to help understand structure.. implementation of each should be conveyable through the docstrings

Reply:
instead something more usable would be a section that described the rds data source

```
### Aurora Severless

// Description of service
To add a Amazon Aurora Serverless cluster as a datasource, use the...

// Example of usage
```ts
usage..
.```

// Note key api needs
Note: A Secret is needed for ...
```
but i defer to @MrArnoldPalmer for final say on this

Reply:
Yeah we have been piling on every new data source example into this example but starting to separate sections out probably makes sense.